src/metrics/jsd/config.vsh.yaml

```
@@ -0,0 +1,32 @@
+__merge__: ../../api/comp_metric.yaml
+
+name: jsd
+info:
+  metrics:
+    - name: jsd
+      label: JSD
```

Is JSD a commonly known acronym for `Jensen-Shannon Distance`?

Otherwise I'd propose using a longer version of the name

```suggestion
    - name: jensen_shannon_distance
      label: Jensen-Shannon Distance
```